docs/tokenomics/nns/proposal-requirements.md

```diff
@@ -0,0 +1,206 @@
+# Submitting a Proposal
+
+## Overview 
+
+### Submit a Proposal
+
+Only neurons with at least 10 ICP and 6 months of dissolve delay can submit proposals to the NNS. To learn more on how to submit a proposal, refer to the [How to Create an NNS motion proposal](https://wiki.internetcomputer.org/wiki/How-To:_Create_an_NNS_motion_proposal). 
+
+### Proposal Format
+
+Each proposal submitted to the NNS has the following fields:
+
+* **Summary:** text providing a short description of the proposal, composed using a maximum of 280 bytes.
+* **URL:** the web address of additional content required to evaluate the proposal, specified using HTTPS. For example, the address might describe content supporting the assignment of a DCID (data center id) to a new data center.
+* **Proposer:** the ID of the neuron that submitted the proposal. When a proposal is submitted, a “charge” is placed on its balance in case it is rejected. So the balance needs to be big enough to pay the charge on (all) rejection(s). We require a neuron to have a dissolve delay of ≥ 6 months to vote, and this applies to submitting proposals too.
+* **Proposal Type:** the type of the proposal. This infers what topic it belongs to (e.g., #NodeAdmin), the system function that will process the proposal if it is adopted, and the type and structure of the parameters that will be passed to that function.
+* **Parameters:** the parameters that will be passed to the system function that will be invoked if the proposal is adopted, as determined by its type. When a proposal is submitted, the NNS checks the parameters.
+
+The NNS assigns a unique identity to each proposal that it receives.
+
+### Topics
+
+Each type of proposal belongs to a specific “proposal topic,” such as “#NodeAdmin” or “#NetworkEconomics,” which determines details about how it will be processed. To prevent users (neurons) from spamming the NNS with proposals, a fee is levied on the neuron that submitted a proposal if it is rejected.
+Proposals are adopted or rejected based on voting activity by “neurons” that network participants have created.
+
+For example, the NNS may require voters to have a greater degree of agreement or to try to process proposals faster, for some topics. Also, neurons follow other neurons on a per-topic basis. Initial topics include:
+
+* **#NeuronManagement:** a special topic using which a neuron can be managed by the followees for this topic (in this case, there is no fallback to default). Votes on this topic are not included in the voting history of the neuron. For proposals on this topic, only the neuron’s followers on the topic that the proposals pertain to are allowed to vote. Because the set of eligible voters for proposals on this topic is restricted, proposals on this topic have a shorter than normal voting period.
```

Comment:
```suggestion
* **#NeuronManagement:** a special topic with which a neuron can be managed by the followees for this topic. Votes on this topic are not included in the voting history of the neuron. For proposals on this topic, only the neuron’s followers on the topic that the proposals pertain to are allowed to vote. Because the set of eligible voters for proposals on this topic is restricted, proposals on this topic have a shorter than normal voting period.
```
How about putting this to the very end as this is irrelevant to most users? 
I didn't understand "(in this case, there is no fallback to default)" - so suggest to remove it